### PR TITLE
withdraw-packages: Withdraw obsolete python3

### PR DESCRIPTION
**Pull request by xnox**
Foundations is working on providing co-installable multiple versions of python (i.e. python-3.11 and python-3.12), such that one can generate wheels / venv of the desired major python version. And we are also working on providing a way to choose which python is default (python-3.11-default, which makes python3 point at python3.11, ditto python-3.12-default, etc).

I have noticed that in the past there was just the one and only python3, and it is still published as installable.

Upon package renames and splits, they should really be unpublished, as otherwise they remain installable and vulnerable.

Fixes: https://github.com/wolfi-dev/os/commit/635a3c353cbfb672bb23ba123ee79bc4dea06bb7

CC: @rawlingsj @smoser @ajayk

### CODE REVIEW

**Review by smoser — DISMISSED** [review]

File: withdrawn-packages.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-pgvector-0.6.1-r0.apk
\ No newline at end of file
+pgvector-0.6.1-r0.apk
+python3-3.11.1-r3
```
Comment (ajayk) [inline withdrawn-packages.txt:2]:
```suggestion
python3-3.11.1-r3.apk
```

File: withdrawn-packages.txt [hunk 2]
```diff
@@ -1 +1,4 @@
...
+python3-dev-3.11.1-r3
+python3-doc-3.11.1-r3
```
Comment (ajayk) [inline withdrawn-packages.txt:4]:
```suggestion
python3-dev-3.11.1-r3.apk
python3-doc-3.11.1-r3.apk
```

File: withdrawn-packages.txt (hunk 1 above)
Comment (smoser) [reply withdrawn-packages.txt:2]: this does look correct, given pgvector having the .apk on the end.
@xnox was there a reason you did not add .apk to the new entriews?

**Comment by xnox** [comment]
@smoser @ajayk was paying too much attention to copy exact package name/version/epoch from apk output and totally forgot about adding .apk extension. Thanks for catching this.

**Review by ajayk — APPROVED** [review]

**Pull request merged**